service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -365,11 +370,15 @@ private Map<String, String> getDefaultParameters() {
       defaultValues.put(ParametersNames.POSTGRES_DB_ADMIN.name(), "db_admin");
       defaultValues.put(ParametersNames.POSTGRES_DB_PASSWORD.name(), UUID.randomUUID().toString());
       defaultValues.put(ParametersNames.POSTGRES_SERVER_SKU.name(), "GP_Gen5_2");
-      defaultValues.put(ParametersNames.VNET_ADDRESS_SPACE.name(), "10.1.0.0/27");
-      defaultValues.put(Subnet.AKS_SUBNET.name(), "10.1.0.0/29");
-      defaultValues.put(Subnet.BATCH_SUBNET.name(), "10.1.0.8/29");
-      defaultValues.put(Subnet.POSTGRESQL_SUBNET.name(), "10.1.0.16/29");
-      defaultValues.put(Subnet.COMPUTE_SUBNET.name(), "10.1.0.24/29");
+      defaultValues.put(ParametersNames.VNET_ADDRESS_SPACE.name(), "10.1.0.0/25");
```

Comment:
cc @sergiygetlin 
Per https://broadworkbench.atlassian.net/browse/WOR-711 I think we're proposing to keep the LZ defaults as-is, but override them from Rawls.

Reply:
Yes. I think this is what we want to achieve. We don't want to change those defaults values. Since they can be overridden, we expect that an upstream service will pass appropriate values.